lib/routes/nhk/news-web-easy.ts

```diff
@@ -33,11 +83,12 @@ export const route: Route = {
 };
 
 async function handler(ctx) {
-    const data = await ofetch('https://www3.nhk.or.jp/news/easy/news-list.json');
-    const dates = data[0];
+    await fetchNews('https://news.web.nhk/tix/build_authorize?idp=a-alaz&profileType=abroad&redirect_uri=https%3A%2F%2Fnews.web.nhk%2Fnews%2Feasy%2F&entity=none&area=130&pref=13&jisx0402=13101&postal=1000001');
+    const response = await fetchNews('https://www3.nhk.or.jp/news/easy/news-list.json');
```

Comment:
Use the redirected URL instead https://github.com/DIYgod/RSSHub/actions/runs/19206023447/job/54901329404#step:13:23